src/fireedge/src/server/routes/api/2fa/index.js

```diff
@@ -14,48 +14,22 @@
  * limitations under the License.                                            *
  * ------------------------------------------------------------------------- */
 
-const { addFunctionAsRoute, setFunctionRoute } = require('server/utils/server')
-const { routes: tfaRoutes } = require('./tfa')
-const { TFA } = require('./string-routes')
+const { Actions, Commands } = require('server/routes/api/2fa/routes')
+const { setup, qr, del } = require('server/routes/api/2fa/functions')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected that you are importing a module inside a function, which is known as lazy loading. If modules are imported within a function, it might prevent other requests from being handled at a more critical time. We recommend that you load all modules at the beginning of each file, before and outside of any functions.